js/views/modals/Settings/Page.js

```diff
@@ -309,6 +311,10 @@ export default class extends baseVw {
           this.headerCropper.cropit('imageSrc',
             app.getServerUrl(`ipfs/${this.profile.get('headerHashes').get('original')}`));
         }
+
+        installRichEditor(this.$('#settingsAbout'), {
```

Comment:
@jjeffryes Can you take the init of the trumbo editor outside of the timeout? It doesn't need to be in one and having it in one breaks the auto-scroll to the first error (when saving with a client side error). I just fixed a similar issue on the Edit Listing modal.